.gitignore

```diff
@@ -0,0 +1,183 @@
+.DS_Store
```

Comment:
I think `.gitignore` should be one. Duplicated `.gitignore` in the project root might cause a problem on another study you will add in the future.